apps/worker/src/app/workflow/usecases/subscriber-job-bound/subscriber-job-bound.usecase.ts

```diff
@@ -95,6 +90,7 @@ export class SubscriberJobBound {
       source: command.payload.__source || 'api',
       subscriberSource: _subscriberSource || null,
       requestCategory: requestCategory || null,
+      statelessWorkflow: !!command.bridge?.url,
```

Comment:
added indication for us to use in the metrics so would know if the trigger was stateless or not.